### PR TITLE
nolo tlc ciel 9_4

### PR DESCRIPTION
**Pull request by nolo16**
sorry for delay on this one, got a bit busy with irl stuff

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 -- Page 10011, Offset 23105.
 -- あの部屋では話にならないでしょう、というノエル先生の提案だった。
+// rule one of torture club
 Noel-sensei suggested that we not speak of what happened in that room.
 }
 [sha:872ba0780913513ef11aaf78ea301a484987165f]{
 -- Page 10012, Offset 23106.
 -- 外気を吸った瞬間、あれほど冷え切っていた感情が元の自分に戻っていた。
-As soon as I breathe in the air of the outside, I feel the glacial emotional state I had been in be thawed back to normal.
+As soon as I breathe in the open air, I feel my frigid emotions returning to their usual state.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:15]: "fresh air" would be my choice just from context, even if 外気 is the word

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 2]
```diff
@@ -211,92 +212,92 @@ I just wanted to startle you and see how you'd react, but considering your react
...
 -- Page 399, Offset 23162.
 -- それに……不思議なコトに、先輩を<怪物|モンスター>と説明したノエル先生の声は卑下するものではなく、むしろ尊敬の念の方が強かった気がする。
-Besides, strangely, I don't feel like Noel-sensei is looking down on Senpai when she's calling her a monster, on the contrary, her tone feels more reverent, if anything.
+Besides, strangely enough, I don't feel like Noel-sensei is looking down on Senpai when she calls her a monster. On the contrary, I feel like her tone was more reverent, if anything.
 }
 [sha:122b680ce0f57eedc50593654933f78dcc953681]{
 -- Page 400, Offset 23163.
 -- 「あ、埋葬機関っていうのは<聖堂教会|わたしたち>にとって、そうね……教会の中の治外法権、権力に近寄れないかわりに権力に縛られない、動く武器庫みたいなものと思って。
-"Ah, the Burial Agency is, for us of the Church... Mmm, think of it like a walking armory, one granted the right to be outside the jurisdiction of the Church, unbound by any authority in exchange from not being able to yield any authority.
+"Ah, right. To us at the Church, the Burial Agency is... mmm, think of it as some sort of walking armory. They're outside the jurisdiction of the Church, and in exchange for not possessing any authority, they themselves are not bound by it.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:300]: ugh, I can't find it but I think 武器庫 might be a lore term. I vaguely remember it coming up in gpr. Probably worth checking with the lore council.

"walking armory" makes it sound like they provide weapons for others, but I think he's more saying like "they have so much firepower, they may as well be an armory" so it should get a light reword

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 3]
```diff
@@ -454,18 +456,18 @@ You're also an Executor, Sensei, so aren't you the same as Senpai?"
...
 -- 「わたしが？　冗談やめて、あんなのと一緒にしないで。
-"Me? Please don't joke around, don't lump me with her.
+"Me? Please stop messing around. Don't lump me in with her.
 }
 [sha:783b03a0aed06690b286526549696e8d5510c901]{
 -- Page 412, Offset 23195.
 -- わたしはいたって普通の人間よ。あんな風にまわりを騙せないし、割り切るコトだってできない」
 // 割り切る make a clean distinction (between her Executor life and her pretend school girl life) without being influenced by personal emotions
-I'm a very normal human. I don't go around deceiving those around me like she does and I can't make such a cold calculus."
+I'm as normal as it gets. I can't just deceive the people around me as she does, and I can't even make a clear distinction either."
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:465]: maybe something more like
I can't just deceive the people around me or keep my work and personal life so separate.

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 4]
```diff
@@ -840,42 +842,43 @@ To begin with, the biggest discrepancy...
...
 -- Page 439, Offset 23271.
 -- 先輩は当たり前のようにシエルと呼ばれていたけど、それ以外の名前を誰も口にしなかった。
-Senpai is referred to as Ciel like it's evident, but nobody has ever called her by any other name.
+Everyone calls her Ciel like it's only natural, but no one has ever referred to her by any other name.
 }
 [sha:a8dc1fe4c9b7e8594f0ab61e649f7a0f381868ee]{
 -- Page 439, Offset 23272.
 -- そんな名前に、どうして誰も疑問を持たなかったのか―――
-Why has no one ever had any suspicions toward that name?
+// is this like 'how come no one ever suspected her when she has a name like ciel?'
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:850]: yes

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 5]
```diff
@@ -1025,13 +1028,13 @@ A part of me harbors doubt that she'll truly be able to erase only this week's m
...
 -- 「そうそう、そういう顔してもらわないと危険を冒した甲斐がないわ。
-"Yup, if you're making such a face, then taking this risk's been worth it.
+"Yup, seeing that look on your face makes all the risk I went through worth it.
 }
 [sha:d81266e50907697fc64bdd099f752170bb24fb0a]{
 -- Page 453, Offset 23309.
 -- それじゃあね志貴クン。今日の話をちゃあんと胸に留めて、わたしみたいに、何もかも台無しにされないよう気をつけて♡」
-// Does わたしみたいに apply to 気をつける or 台無しにされる?
-Well then, Shiki-kun, take to heart what I told you today and take care, just like me, not to have everything ruined♡"
+// Does わたしみたいに apply to 気をつける or 台無しにされる? i'm assuming the latter, but i dont have any particular reason so feel free to correct if you disagree
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:1036]: i mean, the former doesn't really make any sense, does it?

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 9/09_01_CIEL09_4.txt (hunk 2 above)
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_4.txt:300]: searched it up in script and this is the only instance of 武器庫 i saw, but ill check anyway just in case

**Pull request merged**